elastic/apm-server
# Add `source.nat.ip`

## Motivation/summary

as per the ECS team's recommendation, store the address provided in proxy headers `forwarded`, `x-real-ip`, `x-forwarded-for` under `source.{ip,port}` and `client.{ip,port}`; store the source ip addr in `source.nat.ip`

## Checklist




## How to test these changes

1. start apm-server
2. ingest events with an `X-Forwarded-For` header
3. confirm the document has `source.nat.ip` fields

## Related issues

closes https://github.com/elastic/apm-server/issues/7029
